### DISCUSSION TITLE
Schedule Automatic Episode Downloads

### DISCUSSION THREAD

**Discussion by JBHorne2** [advplyr/audiobookshelf#1725] [General]
It appears the option to "Schedule Automatic Episode Downloads" is set per Podcast. Is there any way to change the default here to every hour? By default, it's set to "Custom daily/weekly" which is far too infrequent.

**Comment by advplyr** [comment]
It already defaults to hourly

**Reply by JBHorne2** [reply]
Interesting. How could I verify that? This is what I see for the default on all of my podcasts:

![image](https://user-images.githubusercontent.com/122175763/234595241-dfdfe2e6-7a02-45ea-8947-a6e44c5dc5e4.png)

And the Advanced tab:

![image](https://user-images.githubusercontent.com/122175763/234595501-0054607e-e99b-4520-9053-51944a20d491.png)

**Reply by advplyr** [reply]
Hmm this may be a bug. I'll check it out
